modules/widgets/my-community.php

```diff
@@ -36,7 +36,7 @@ function __construct() {
 			/** This filter is documented in modules/widgets/facebook-likebox.php */
 			apply_filters( 'jetpack_widget_name', esc_html__( 'My Community', 'jetpack' ) ),
 			array(
-				'description' => esc_html__( 'A sampling of users from your blog.', 'jetpack' ),
+				'description' => esc_html__( 'Display members of your site\'s community.', 'jetpack' ),
```

Comment:
IMHO, the string should be double quoted so the word is simpler and users don't have to deal with `\`, which in other circumstances is used before a control character.